torchfields/fields.py

```diff
@@ -887,6 +887,18 @@ def __rpow__(self, other):
     def gaussian_blur(self, sigma=1, kernel_size=5):
         return voting.gaussian_blur(self, sigma, kernel_size)
 
+    @wraps(voting.vote)
+    def get_vote_shape(self):
+        return voting.get_vote_shape(self)
+
+    @wraps(voting.vote)
+    def get_vote_subsets(self):
+        return voting.get_vote_subsets(self)
+
+    @wraps(voting.vote)
+    def get_vote_weights(self, softmin_temp=1, blur_sigma=1):
+        return voting.get_vote_weights(self, softmin_temp, blur_sigma)
```

Comment:
The [`@wraps` decorator](https://docs.python.org/2/library/functools.html#functools.wraps) converts a wrapping function to look like the wrapped function, including adopting the wrapped function's documentation.
So for each one of these, the argument to the decorator should be a reference to the function being wrapped (that is, instead of all of them receiving their documentation from `voting.vote`).